scalatags/shared/src/main/scala/scalatags/generic/Util.scala

```diff
@@ -43,12 +43,13 @@ trait Util[Builder, Output <: FragT, FragT] extends LowPriUtil[Builder, Output,
     /**
      * Converts the string to a [[Attr]]
      */
-    def attr = {
+    def attr: Attr = attr(None)
+    def attr(namespace: Option[Namespace]): Attr = {
```

Comment:
In theory: setAttribute(name, value) == setAttributeNS(null, name, value). With implicit namespaceConfig parameter it will be very easy to break it by accident.